packages/release-trigger/cloudbuild.yaml

```diff
@@ -45,15 +47,31 @@ steps:
       - "$_KEY_RING"
       - "$_FUNCTION_REGION"
 
+  - name: gcr.io/cloud-builders/docker
+    id: "build-docker-frontend"
+    waitFor: ["-"]
+    dir: packages/release-trigger
+    args:
+      - "build"
+      - "-f"
+      - "Dockerfile.frontend"
+      - "-t"
+      - "gcr.io/$PROJECT_ID/release-trigger-frontend"
+      - "."
+
+  - name: gcr.io/cloud-builders/docker
+    id: "push-docker-frontend"
+    waitFor: ["build-docker-frontend"]
+    args: ["push", "gcr.io/$PROJECT_ID/release-trigger"]
```

Comment:
It needs to be `release-trigger-frontend`

Reply:
Thanks, good catch